go.mod

```diff
@@ -235,7 +235,7 @@ replace (
 	github.com/Parallels/docker-machine-parallels/v2 => github.com/minikube-machine/machine-driver-parallels/v2 v2.0.1
 	github.com/briandowns/spinner => github.com/alonyb/spinner v1.12.7
 	github.com/docker/machine => github.com/minikube-machine/machine v0.0.0-20230610170757-350a83297593
-	github.com/machine-drivers/docker-machine-driver-vmware => github.com/minikube-machine/machine-driver-vmware v0.1.5
+	github.com/machine-drivers/docker-machine-driver-vmware => github.com/lbogdan/docker-machine-driver-vmware v0.2.0
```

Comment:
@lbogdan do you mind making this PR to the new Org https://github.com/minikube-machine/machine-driver-vmware ?

Reply:
Yes, I will get to it on Friday, weekend at the latest.

Reply:
For getting the release out in time we'll merge this and then change it back to our fork once the change gets merged upstream.

Reply:
Created https://github.com/machine-drivers/docker-machine-driver-vmware/issues/75